installbuilder/datafiles/Base_DSC.data

```diff
@@ -489,6 +489,18 @@ chown -R omsagent /opt/microsoft/${{SHORT_NAME}}/Scripts
 su - omsagent -c "/opt/microsoft/${{SHORT_NAME}}/Scripts/ImportGPGKey.sh /opt/microsoft/${{SHORT_NAME}}/keys/msgpgkey.asc keymgmtring.gpg"
 su - omsagent -c "/opt/microsoft/${{SHORT_NAME}}/Scripts/ImportGPGKey.sh /opt/microsoft/${{SHORT_NAME}}/keys/dscgpgkey.asc keyring.gpg"
 
+### Reset ownership of files defined in /etc/sudoers.d/omsagent
```

Comment:
I think some scripts in [sudoers](https://github.com/microsoft/OMS-Agent-for-Linux/blob/master/installer/conf/sudoers) might be missing, e.g. 

```
/opt/microsoft/omsconfig/Scripts/OMSSyslog-ng.post.sh
/opt/microsoft/omsconfig/Scripts/NPMAgentBinaryCap.sh
/opt/microsoft/omsconfig/Scripts/OMSAuditdPlugin.sh
```

Reply:
I will add OMSSyslog-ng but we don't own the following:
/opt/microsoft/omsconfig/Scripts/NPMAgentBinaryCap.sh
/opt/microsoft/omsconfig/Scripts/OMSAuditdPlugin.sh